keyboard/.htaccess

```
@@ -1,18 +1,14 @@
 
-# index stop processing
-RewriteRule "^index.php$" - [L]
-
 # Redirect broken legacy keyboard links
-RewriteRule "^gff_(mdx|muz|suq)_7(.*)" "gff_mym_7$2" [R]
+RewriteRule "^gff_(mdx|muz|suq)_7(/.*)?" "/keyboard/gff_mym_7$2" [R,L]
 
 
 # Rewrite /keyboard/[name][/] to /keyboard/redir.php?q=[name]
 # only if index.php does not exist in the folder
-RewriteCond "%{DOCUMENT_ROOT}/keyboard/(?!redir.php)$1/index.php" !-f
-RewriteRule "/keyboard/([^/?]+)(/)?$" "/keyboard/redir.php?q=$1" [R,L]
-
+RewriteCond "%{DOCUMENT_ROOT}/keyboard/$1/index.php" !-f
+RewriteRule "^(?!index.php|redir.php)([^/?]+)(/)?$" "/keyboard/redir.php?q=$1" [END]
```

```suggestion
RewriteRule "^(?!index.php|redir.php)([^/?]+)(/)?$" "/keyboard/redir.php?q=$1" [R,L]
```

This was a redirect, I think it still needs to be